### PR TITLE
do not load loomio_org plugins in Dockerfile

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <loic@dachary.org>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ ENV DEVISE_SECRET boopboop
 ENV SECRET_COOKIE_TOKEN beepbeep
 
 # build assets
-RUN bundle exec rake deploy:build
+RUN bundle exec rake deploy:build docker
```
Comment (gdpelican) [inline Dockerfile]: This is definitely the right approach, but I found I needed 

```
RUN bundle exec rake deploy:build[docker]
```

to make this go. Does passing `docker` in like that work for you?

File: plugins/docker.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Plugins loaded by https://github.com/loomio/loomio/blob/master/Dockerfile
+plugins:
```
Comment (gdpelican) [inline plugins/docker.yml]: This wants to mirror https://github.com/loomio/loomio/blob/master/config/plugins.yml
(we've dropped the top level 'plugins' line)

File: Dockerfile (hunk 1 above)
Comment (dachary) [inline Dockerfile]: I did not try, that was a quick draft after browsing the code, to check if I was going in the right direction :-)

File: plugins/docker.yml (hunk 2 above)
Comment (dachary) [inline plugins/docker.yml]: ok

**Comment by dachary** [comment]
amended to address comments & repushed

**Comment by gdpelican** [comment]
Sweet. I've made a few further changes we need to the plugin system here: #3385 

Closing this one for now; we'll get that one merged in soon. Thanks for this!

**Pull request closed without merging**